### ISSUE TITLE
suggestion - remove wallet number and icon without leaving a big ass gap

### ISSUE THREAD

**Issue by kingofcube** [anthony1x6000/ROBLOX2016stylus#51]
please im begging you

**Comment by anthony1x6000** [comment]
Like remove the element that shows how much robux you have?

**Comment by anthony1x6000** [comment]
You can just do ```#navbar-robux {
    display: none;
}```

**Comment by kingofcube** [comment]
> Like remove the element that shows how much robux you have?

No, I meant the roblox credit bullshit

**Comment by anthony1x6000** [comment]
I am not familiar, could you show me a screenshot?

**Comment by GoofyGuyWhoCantAnimate** [comment]
Roblox does not have an official credit converter, you must be using other extensions. Just disable it that way

**Comment by anthony1x6000** [comment]
Probably a third-party extension. I'll keep this issue closed until the extension is named so I can add support for it then.

**Comment by kingofcube** [comment]
> Probably a third-party extension. I'll keep this issue closed until the extension is named so I can add support for it then.

![floorp_NoKNWdW9N2](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/138169081/74137f29-1b13-4a4f-a585-4bc77db333b4) ????? i just disabled the roblox addons i had and it still shows up

**Comment by anthony1x6000** [comment]
I have never seen that in my life, but [it seems part of roblox](https://www.reddit.com/r/RobloxHelp/comments/18qr2wm/im_a_bit_confused_what_is_this_wallet_thingy_and/).

Unfortunately, I cannot fix this as I don't yet have that icon. Could you inspect that element and send a screenshot -  similar to the following image?
![image](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/33004321/5a5f28b5-2480-42b9-b293-9fa190deb8d1)
I could maybe try fixing it that way, otherwise hopefully someone can fix it themselves and make a pull request.

**Comment by parker02311** [comment]
@anthony1x6000 
This can be removed by adding the following CSS
```css
    /* Remove nav-credit display */
    .nav-credit {
        display: none !important;
    }
```

What it looks like without
![image](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/35582919/4b5140b5-ac88-41a3-8e28-94bc4afd2e6a)

What it looks like with:
![image](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/35582919/87b3d831-bcfc-41ff-8047-1728355980e5)

**Comment by anthony1x6000** [comment]
That's great! 

If possible, could you also test the icon colours on light and dark modes?
 
It may also be worth making an optional toggle similar to 'displayRecommended.' I could do that part myself to save you the trouble.

**Comment by parker02311** [comment]
I have already implemented the option toggle! Creating the PR right now actually, I will quickly test the icon colors, although I would have no idea how to fix them if they are broken.

**Comment by anthony1x6000** [comment]
Wow, thanks for already implementing that!
Should the icons be broken, sometimes I just do a css filter invert(1) on them. Probably not the best practice. 
Sometimes the light and dark mode icon is in the same like svg, so just moving the background image might work as well.

**Comment by parker02311** [comment]
Looking into it, they are broken but Roblox does the color changing for this icon by applying the filter invert(1), currently trying to find a way to override it.

**Comment by parker02311** [comment]
It appears it is broken, but I have found a fix and will add it to the PR

**Comment by anthony1x6000** [comment]
Should be fixed now in #52
